packages/credentials/src/ctype/CTypeLoader.ts

```diff
@@ -5,13 +5,15 @@
  * found in the LICENSE file in the root directory of this source tree.
  */
 
-import { ICType } from '@kiltprotocol/types'
+import type { ICType } from '@kiltprotocol/types'
+import { SDKErrors } from '@kiltprotocol/utils'
 
 import { fetchFromChain } from './CType.chain.js'
+import { isICType, verifyDataStructure } from './CType.js'
 
 export type CTypeLoader = (id: ICType['$id']) => Promise<ICType>
 
-const loadCType: CTypeLoader = async (id) => {
+const chainCTypeLoader: CTypeLoader = async (id) => {
```

Comment:
Not entirely sure why the name change `loadCType` sounded better. I think people would understand that more.

I understand this fetches from the chain. But should this matter in the eyes of a user?

Reply:
I suggest we revert this. If I wanted to get a ctype I would have a hard time to find this.

The action first then subject second makes more sense, considering the user might fully understand the subject but the action is easy to grasp.

Reply:
I can't tell you exactly why I renamed this, probably because it hated the look of the default argument definition
```
cTypeLoader = loadCType
```
Doesn't really tell you much, while `chainCTypeLoader` at least explains where the CTypes are loaded from.
That said, besides maybe showing up in the function signature as the default argument, this does not come up in user land as it is not exported from this file. But if you hate it I can revert.

Reply:
Hmm, but it seems worse to me now `CTypeLoader.chainCTypeLoader`. 

Let us put this into the dev channel and see.

Reply:
again, it's not exported, so there's neither `CTypeLoader.chainCTypeLoader` nor `CTypeLoader.loadCType`. It's not accessible from the outside at all